jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -726,23 +739,24 @@ def _update_stretch_histogram(self, msg={}):
 
         # update the n_bins since this may be a new layer
         self._histogram_nbins_changed()
+        # update the curve/colorbar

```

Comment:
I don't think colorbar and stretch curve listen to exactly the same events? For example, colorbar needs to match the bins on histogram but not necessarily the curve?

Reply:
If there are blocks you think we should skip under certain scenarios, please suggest any changes (`msg.get('name') should give the name of the traitlet causing the method to be triggered)!